app/registries/foreman/plugin/fact_parser_registry.rb

```diff
@@ -0,0 +1,24 @@
+module Foreman
+  class Plugin
+    class FactParserRegistry
+      attr_reader :parsers
```

Comment:
Is this really something we want to expose? I don't think it's needed with the API that's already there.

Reply:
It was used in tests. I updated the code of test and added `unregister(key)` method to registry for proper test working.